core/flyout_button.js

```diff
@@ -227,6 +227,13 @@ Blockly.FlyoutButton.prototype.moveTo = function(x, y) {
   this.updateTransform_();
 };
 
+/**
+ * @returns {boolean} Whether or not the button is a label.
```

Comment:
There's a really good reason for this, and I think it has something to do with our TypeScript generation, but we use @return instead of @returns in Blockly. Doing a quick search, looks like it's there in three other places right now.

Reply:
This didn't feel right but VSCode autofilled it for me and I trusted it :(